### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Agarwalvidu** [community/community#112263] [GitHub Education]
### Hi everyone,

Greetings to all my dear fellow GitHub Global Campus Students and our teachers.
Myself **Vidushi Agarwal** and my GitHub profile ID is [https://github.com/Agarwalvidu](url), I am currently pursuing 2nd year in B.Tech from IGDTUW. I am very much interested into web development (including frontend and backend), keeping my interest on I am also mentoring my juniors in the same field. I am also exploring the world of 3D web development where I am learning ThreeJs, WebGL and a tool called blender.
I hope to have a great experience here.

Warm Regards,
Vidushi Agarwal

**Comment by drdixit** [comment]
Just dropped by to say 👋Hello! @Agarwalvidu
